source-analysis.md

```diff
@@ -1,36 +1,47 @@
-
-Overall, your answers should be *roughly* the equivalent of 2 pages, double-spaced.
-
-See the [README.md](README.md) file for information on how to submit this assignment.
+# Source analysis (Stephen DeCasien)
 ---
-
 ## Basic Source Information
 
 * Where is your source located (URL? Or traditional media)
+http://www.obelisks.org/en/index.htm
+
 * What kind of source is it? (Diaries, records of objects, data tables, etc...)
+The website is a summary of the information and data on each obelisk worldwide, based on this persons own travels. It has a list of obelisks that are linked to their present site with a multitude of information concerning each obelisk.
+
 * Who, or what organization, is responsible for hosting the source (if online), who gathered the information?
+This was a personal project of two "obelisks fans". There names are Hiroyuki Nagase and Shoji Okamoto. It seems that Hiroyuki Nagase did most of the work visiting each obelisk and collecting the data about them.
 
 ---
 
 ## Authorship
 
 * Is there any information available discussing the process by which the information was gathered, or how the digital project was constructed?
-* How transparent is the project about the choices they made in gathering and transforming the data for digitalization? [Click here for an example of a an excellent 'About' page from a database about Anglo-Saxons](http://www.pase.ac.uk/about/index.html)
+The author of the website states that he went to these obelisks to observe and take data on there present condition. It seems that some of the information has been taken from Wikipedia, Google Maps, and linked to different websites related to each individual obelisk.
+
+* How transparent is the project about the choices they made in gathering and transforming the data for digitalization?
+The project seems to be fairly transparent. The author has sections broken down into different discusses about why each obelisk was chosen and how they were studied.  The author describes the short falls of the website right away, saying that it is not complete nor 100% accurate.
+
 * Do the organizations or individuals involved in gathering and publishing your data set have any stated goals or agendas?
+There goal was to collect the most information they could on their travels about the obelisk around the world. They called themselves "obelisks fans".
+
 * Do the organizations or individuals involved seem to have any unstated agendas or goals?
+Just seems to be a bunch of Japanese obelisk enthusiasts.
```

Comment:
Might there be any way to find out more, or to consider how their perspective might affect their view of what constitutes 'the data'?